apps/docs/src/app/core/component-docs/dialog/dialog-docs.component.html

```diff
@@ -230,20 +230,21 @@
 <ng-template let-dialog let-dialogConfig="dialogConfig" #playgroundDialog>
     <fd-dialog [dialogRef]="dialog" [dialogConfig]="dialogConfig">
         <fd-dialog-header>
-            <h1 fd-title>Playground Dialog</h1>
-            <button fd-dialog-close-button (click)="dialog.dismiss()"></button>
+            <h1 fd-title id="fd-dialog-header-11">Playground Dialog</h1>
+            <button fd-dialog-close-button (click)="dialog.dismiss()" i18n-title="@@dialogPlayGround.closebutton.title"></button>
```

Comment:
for the example, i don't think we need to give i18n-title. But, we need to have `title` here instead. 

I think what we discussed was since you cannot add i18n marker to the `host` property of `fd-dialog-close-button`, we should instead add the `title` and `i18n-title` to wherever it is used in the dialog component. It should be in the library components where `fd-dialog-close-button` is used, not in the examples. Translation of examples is something we are not handling as of now so it is not needed here.

Also, you cannot have just `i18n-title` here and `title` in the library. We can't expect user to do this wherever they use `fd-dialog-close-button`, right? Therefore, both `title` and `i18n-title` should come from the library itself. 
Hope this is clear.

Reply:
fixed

Reply:
`i18n-title` is not needed for example files.